tests/orm/nodes/test_repository.py

```diff
@@ -140,6 +140,18 @@ def test_sealed():
         node.base.repository.put_object_from_bytes(b'content', 'path')
 
 
+def test_updatable_objects():
+    node = CalcJobNode()
+    node._updatable_objects = ['not_raise']
+    node.store()
+    node.base.repository.put_object_from_bytes(b'content', 'not_raise')
+    with pytest.raises(exceptions.ModificationNotAllowed):
+        node.base.repository.put_object_from_bytes(b'content', 'raise')
+    node.seal()
+    with pytest.raises(exceptions.ModificationNotAllowed):
+        node.base.repository.put_object_from_bytes(b'content', 'not_raise')
```

Comment:
Hi @unkcpz , the test here provides an example use case. I will give another example of WorkGraph later.

Reply:
If you do:

```python
    node._updatable_objects = ['not_raise']
    node.store()
    node.base.repository.put_object_from_bytes(b'content', 'not_raise')
    node.store()
    node.base.repository.put_object_from_bytes(b'another content', 'not_raise')
```

do you check it will create a new one or mutate the old one? I believe it create new.

Reply:
You are right. It created another file with a different key instead of overriding the old one.